src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import Details from './components/Details';
+import Header from './components/Header';
+import Home from './components/Home';
+
+function App() {
+  return (
+    <div className="App container">
+      <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="details" element={<Details />} />
```

Comment:
- [ ] All your item detail pages have the same route `/details`, but that should be the case. Each item should have its own unique route, for example: `/details/ETHBTC`.

  ![eth](https://user-images.githubusercontent.com/36542986/189099064-894cfce7-a990-4239-94fb-29d6a04e6787.jpg)